packages/components/src/components/Select/Select.tsx

```diff
@@ -135,6 +140,7 @@ export const SelectInput = (
                               'aria-label': ariaLabel,
                               ...(forwardedRef ? { ref: forwardedRef } : {}),
                           }))}
+                    {...(id ? { id } : {})}
```

Comment:
I think we don't need to spread the id this way here we can just do 
```suggestion
                    id={id}
```
no?

Reply:
No, I only want to override the id if it is actually set outside (and I even feel bad about that)

Reply:
![Screenshot 2025-02-19 at 10 20 34](https://github.com/user-attachments/assets/6efeb659-6949-461f-8ddc-64f520c15149)

If you look at the generated html you'll notice the Radix component already take quite a lot of control.